yotpo-demo/src/get-all.ts

```diff
@@ -0,0 +1,15 @@
+import dotenv from 'dotenv';
+
+fetch(process.env.YOTPO_API_URL+'customers/recent?per_page=100',{
+            method: 'GET',
+            headers: {
+                'content-type': 'application/json',
+                'x-api-key': process.env.YOTPO_API_KEY,
+                'x-guid': process.env.YOTPO_GUID,
+            }}).then((response) => {
```

Comment:
Ideally this formatting/indentation would be more consistent... this `.then` is kinda buried here